src/tests/combine.test.js

```diff
@@ -356,4 +356,10 @@ describe("combine", () => {
       ],
     });
   });
+
+  test("Both users have one empty cell", () => {
```

Comment:
Hey @vdav02, just looking at this test here and it seems like this set of inputs is not something that can by produced by the rest of the code, so I'm not sure this test is applicable for this current release. I think it is acceptable to remove this failing test 😄

Reply:
Hmm, not sure how I missed that when I ran jest on my end... I'll have another look now and remove it if I can reproduce the error.